project/profiles/capacity4more/modules/c4m/features/c4m_features_overview_groups/views/handlers/c4m_overview_groups_handler_title_no_purl.inc

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @file
+ * Contains the handler for node's title field.
+ */
+
+/**
+ * When a node title's is linked inside a group context, the URL is modified
+ * by the PURL API. This handler adds the option field to Disable PURL,
+ * and adds the direct link to the node.
+ *
+ * @ingroup views_field_handlers
+ */
+class c4m_overview_groups_handler_title_no_purl extends views_handler_field_node {
+
+  /**
+   * {@inheritdoc}
+   */
+  function option_definition() {
+    $options = parent::option_definition();
+    $options['disable_purl'] = array('default' => isset($this->definition['disable_purl default']) ? $this->definition['disable_purl default'] : FALSE, 'bool' => TRUE);
+    return $options;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  function options_form(&$form, &$form_state) {
+    $form['disable_purl'] = array(
+      '#title' => t('Disable PURL'),
+      '#description' => t("Disable persistent URL from links."),
+      '#type' => 'checkbox',
+      '#default_value' => !empty($this->options['disable_purl']),
```

Comment:
why would `$this->options['disable_purl']` be empty if it has been defined on the `function option_definition()` ?